mobile/src/main/res/layout/uwazi_send_fragment.xml

```diff
@@ -42,29 +42,37 @@
         />
 
     <androidx.appcompat.widget.AppCompatButton
-        style="@style/Unlock_White_Text_Back"
+        style="@style/Unlock_White_Text"
+        android:layout_marginBottom="18dp"
         android:layout_width="0dp"
-        android:layout_height="54dp"
+        android:layout_height="50dp"
         android:id="@+id/cancel_btn"
         android:text="@string/Uwazi_Submit_Later"
+        android:paddingStart="12dp"
+        android:paddingEnd="12dp"
         android:background="@drawable/action_button_background"
         android:backgroundTint="@color/wa_white_20"
+        android:textStyle="bold"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toStartOf="@id/next_btn"
         app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
can we have a before and after screen shots just to make sure that the change is working properly?

Reply:
Probably, but it should work because those changes have already been confirmed by the design team on the alpha version.